app/src/main/res/raw/keyboard_layout_russian_phonetic_linux.kcm

```diff
@@ -0,0 +1,439 @@
+# License: MIT
+
+# Russian Phonetic Linux
+type OVERLAY
+
+map key 2 1
+map key 3 2
+map key 4 3
+map key 5 4
+map key 6 5
+map key 7 6
+map key 8 7
+map key 9 8
+map key 10 9
+map key 11 0
+map key 12 MINUS
+map key 13 EQUALS
+map key 16 Q
+map key 17 W
+map key 18 E
+map key 19 R
+map key 20 T
+map key 21 Y
+map key 22 U
+map key 23 I
+map key 24 O
+map key 25 P
+map key 26 LEFT_BRACKET
+map key 27 RIGHT_BRACKET
+map key 30 A
+map key 31 S
+map key 32 D
+map key 33 F
+map key 34 G
+map key 35 H
+map key 36 J
+map key 37 K
+map key 38 L
+map key 39 SEMICOLON
+map key 40 APOSTROPHE
+map key 41 GRAVE
+map key 43 BACKSLASH
+map key 44 Z
+map key 45 X
+map key 46 C
+map key 47 V
+map key 48 B
+map key 49 N
+map key 50 M
+map key 51 COMMA
+map key 52 PERIOD
+map key 53 SLASH
+map key 57 SPACE
+map key 86 PLUS
+map key 83 NUMPAD_COMMA
+
+key 1 {
+  label: '1'
+  base: '1'
+  shift: '\u0021'
+  capslock+shift: '\u0021'
+}
+
+key 2 {
+  label: '2'
+  base: '2'
+  shift: '\u044a'
+  capslock+shift: '\u044a'
+}
+
+key 3 {
+  label: '3'
+  base: '3'
+  shift: '\u0451'
+  capslock+shift: '\u0451'
+  ralt: '\u2116'
+}
+
+key 4 {
+  label: '4'
+  base: '4'
+  shift: '\u0401'
+  capslock+shift: '\u0401'
+}
+
+key 5 {
+  label: '5'
+  base: '5'
+  shift: '\u044a'
+  capslock+shift: '\u044a'
+}
+
+key 6 {
+  label: '6'
+  base: '6'
+  shift: '\u042a'
+  capslock+shift: '\u042a'
+}
+
+key 7 {
+  label: '7'
+  base: '7'
+  shift: '\u0026'
+  capslock+shift: '\u0026'
+}
+
+key 8 {
+  label: '8'
+  base: '8'
+  shift: '\u002a'
+  capslock+shift: '\u002a'
+}
+
+key 9 {
+  label: '9'
+  base: '9'
+  shift: '\u0028'
+  capslock+shift: '\u0028'
+}
+
+key 0 {
+  label: '0'
+  base: '0'
+  shift: '\u0029'
+  capslock+shift: '\u0029'
+}
+
+key MINUS {
+  label: '\u002d'
+  base: '\u002d'
+  shift: '\u005f'
+  capslock+shift: '\u005f'
+}
+
+key EQUALS {
+  label: '\u003d'
```

Comment:
The label is misleading, because this doesn't type the equals sign, right?

If true, how do you type the equals sign with this layout on Linux? Don't you need it?

Reply:
On Linux one would use the third level key to type equals in this layout, on Android, personally, I don't need it, but I get your point.
I'm not sure if "third level" is something that Android has, so I've looked around in existing layouts and I see that ralt is used in places.
So what do you say if I put equals on ralt, as well as change the label to the actual symbol that is being typed without ralt?

Reply:
I agree, right alt is an option. How does the layer work in Linux exactly? Do you lock the keyboard into the third layer and then unlock? Which modifiers are involved?

Sorry for the late reply, I promise to get back to you within 24h next time

Reply:
Hey, no worries, I was pretty busy these couple of weeks myself.
I'll try to update the PR today, including the answers to your questions about the layout.